wordpress-mobile/WordPress-iOS
# Remove selected_site_self_hosted tag from Zendesk tickets on Atomic sites

Fixes #NA

To test:

Pre-requisite: have an account with at least one atomic site, one self hosted site and one Wp.com site
- Build/run and select the atomic site in My Site
- Open a support request (Me -> Help & Support -> Contact Support)
- Go on Zendesk, look for your ticket in the mobile queue, and make sure that the tags section does not contain the tag `selected_site_self_hosted`
- Repeat the se steps for the self hosted site and make sure that, this time, the tags section contains the tag `selected_site_self_hosted`
- Repeat the steps for the wp.com site and make sure that the tags section does not contain the tag `selected_site_self_hosted`

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
